src/diagonal.works/b6/api/functions/collections_test.go

```diff
@@ -290,3 +292,36 @@ func TestHistogram(t *testing.T) {
 		t.Errorf("Found diff (-want, +got):\n%s", diff)
 	}
 }
+
+func TestExport(t *testing.T) {
```

Comment:
btw we only want export here / we want read in private / re where we ended our chat discussion earlier